### ISSUE TITLE
Install foundationPress twice in one themes directory?

### ISSUE THREAD

**Issue by andreaarcher** [olefredrik/FoundationPress#769]
Hello,
I am developing 2 separate sites on my local MAMP stack. I'd like to use FoundationPress (installed via npm) for both of them. The first install went well, now I want to clone/install again and change the name of the theme so as to not have a conflict, but I'm not sure what the workflow would be, I'm afraid that I will overwrite my first install if I go through the command line again using the original instructions:

$ cd my-wordpress-folder/wp-content/themes/
$ git clone https://github.com/olefredrik/FoundationPress.git
$ cd FoundationPress
$ npm install

Can anyone advise?

Thanks so much in advance

Andi

**Comment by colin-marshall** [comment]
As long as you aren't in the same directory you will be fine.

**Comment by andreaarcher** [comment]
I just renamed my original to "FoundationPressNew" and then did an new clone -- of couse the new one works great but the old one won't compile-- can I throw out the new one and get the old one working again?

**Comment by erwinpagulong** [comment]
try this.
 git clone https://github.com/olefredrik/FoundationPress.git
 <your-theme-name>

it will save to the "your-theme-name"






Erwin B. Pagulong
Front End Web Developer

Mobile: +63 908 7793 773
Web: http://erwinpagulong.com/

**Comment by andreaarcher** [comment]
Ok, changed the original back to FoundationPress and all is well, still not sure I can have 2 versions running from the same folder, not sure how I'm going to develop more than one foudationpress site at a time now. Any suggestions are appricitated, sorry if my question is completely noob

**Comment by JayCadilak** [comment]
I have not had that problem and have never used the default folder name. Just clone or extract the FoundationPress directory, rename the directory to whatever you want, then run npm install from within the directory. That's my procedure and I can install as many as I want within the same WordPress installation with no conflicts. Hope that helps!

**Comment by JayCadilak** [comment]
On a side note from something you wrote, if you are developing two sites you should have two completely separate WordPress installations. Then you can clone or extract the FoundationPress directory into each installation and leave its name as-is without them interfering with eachother.

**Comment by andreaarcher** [comment]
I figured that, it also looks like I can have both in the same folder, only one can actively be compiled, so in
Theory I could switch the name whenever I want to work in one of them. I haven't tested this out yet, but I think it might work. The reson I think this is that once I realized that my renamed original wouldn't compile, I deleted the new install and then changed the name back to foundationPress. Then it worked fine. I think I could've kept the new install in place but just changed the name to foundationPressBAK. I might try it out. I don't know how to do multiple Wordpress installs on myMAMP stack. I'm sure I could figure it out, but if I could just "toggle" between the themes I'm developing by changing the names that might be quicker-

**Comment by Eccola** [comment]
@andreaarcher check out WordPress multisite network: http://codex.wordpress.org/Create_A_Network
You can run it on MAMP just the same way as a single WP install.

**Comment by andreaarcher** [comment]
I use multisite now, but there is still only one theme directory so that won't help in this situation. but thanks so much for the suggestion. 

Ok, thanks the collective genius of this board I've figured this out. As per JayCadilak I can change the name "FoundationPress" to anything I want before I run install. Since my first clone/build was set up as FoundationPress, it will only compile if it is called that. So—— when I want to add a new one I temporarily change the name to something else so that I don't overwrite it when I install the new one via the "git clone https://github.com/olefredrik/FoundationPress.git" command. then I change the name of the new one to something else BEFORE I run the install command. Then I change my original back to FoundationPress and run watch again. I just tested and is seems to be working.
